debs/generate_debian_package.sh

```diff
@@ -45,10 +45,12 @@ build_deb() {
     CONTAINER_NAME="$1"
     DOCKERFILE_PATH="$2"
 
+
```

Comment:
This line is unnecessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/bc452d8215b7b0d0fc9e9d8e96fa88305af7110a.